### ISSUE TITLE
youtube 404 errors Continues... in 2.1.2

### ISSUE THREAD

**Issue by appysrinath** [streamlink/streamlink#3799]
## Issue

- [ ] This is not a bug report, feature request, or plugin issue/request.
- [x] I have read the contribution guidelines.
- [x] I am using the latest development version from the master branch.

### Reproduction steps / Explicit stream URLs to test

1. Use Any Youtube Link ...
2. ...
3. ...

### Log output

```
C:\Program Files (x86)\Streamlink\bin>streamlink.exe https://youtu.be/88uwWAVyvPY best
[cli][info] Found matching plugin youtube for URL https://youtu.be/88uwWAVyvPY
error: Unable to open URL: https://youtube.com/get_video_info (404 Client Error: Not Found for url: https://www.youtube.com/get_video_info?video_id=88uwWAVyvPY&html5=1&el=detailpage)
```

### Additional comments, etc.

[Love Streamlink? Please consider supporting our collective. Thanks!](https://opencollective.com/streamlink/donate)

**Comment by yavuzKomecoglu** [comment]
[this was resolved with pull request](https://github.com/streamlink/streamlink/pull/3797) .  No new release version has been released yet.

**Comment by bastimeyer** [comment]
Duplicate of #3795
Resolved by #3797

> * I am using the latest development version from the master branch.
> https://youtube.com/get_video_info

You are **not using the latest version from the master branch**.

https://streamlink.github.io/latest/install.html#pypi-package-and-source-code

There will be a `2.2.0` release soon.

**Comment by bastimeyer** [comment]
`2.2.0`:
https://github.com/streamlink/streamlink/releases/latest
https://streamlink.github.io/changelog.html#streamlink-2-2-0-2021-06-19
